.github/workflows/build-legacy.yml

```diff
@@ -1,4 +1,4 @@
-name: Build
+name: Build legacy image
```

Comment:
I think we can drop those old workflows. For people using the `:latest` image we can ask them to use the new tag `public.ecr.aws/spacelift/runner-ansible:7` to have the latest security fixes.